### ISSUE TITLE
Symbols list quickly becomes cluttered

### ISSUE THREAD

**Issue by Azeirah** [cdglabs/apparatus#20]
When there are too many symbols in the list, the view itself becomes obscured by the scrollbar.
I'd like to show a screenshot of this happening, but I can't upload from my current location. Not sure if this is an issue in browsers other than Firefox.

Anyhow, even if it weren't obscured, working with just a few symbols very quickly becomes cumbersome.

By default, the symbols take up a _lot_ of white-space, perhaps we could halve their size? Halve the size of the symbol containers, and scale up the symbols themselves to fit the rectangles, approximately four times as many symbols will fit in the same view!

**Comment by electronicwhisper** [comment]
Yes, agreed that this panel could use some design cleanup. Would be happy to discuss mocks if anyone wants to take a crack at it.

One thing that may help that I never got around to is an algorithm to determine the bounding box of a shape. Then this could be used to "crop" the thumbnails so that there's less whitespace.
